### PR TITLE
improve SEO on statements

### LINKED ISSUE #32497: Improve SEO based on keywords, operators and tokens
The project to bring coalesce multiple pages for statements, operators, keywords, and tokens has created a better reading experience. However, in many bases it's been at the cost of SEO for the language keywords. 

Most of this can probably be addressed by adding the keywords to the title and description data, and the H1 heading and introductory paragraphs. These additions should also add "C#" to the text. The keywords themselves are common enough English words that they don't have great SEO value. (e.g. "C# if" vs. "if").

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: a60b6e17-36cf-bd6c-1b76-d38f1760a816
* Version Independent ID: 0fdeaa81-c3c2-c290-224e-792fcee9b297
* Content: [C# reference](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/)
* Content Source: [docs/csharp/language-reference/index.md](https://github.com/dotnet/docs/blob/main/docs/csharp/language-reference/index.md)
* Product: **dotnet-csharp**
* Technology: **csharp-language-reference**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

---
[Associated WorkItem - 34653](https://dev.azure.com/msft-skilling/Content/_workitems/edit/34653)

### PR DESCRIPTION
**Pull request by BillWagner**
In all the statements, operators, and tokens pages, edit the title, description, and opening paragraph to increase SEO on the statement keywords.

First commit - edit all statements pages.
Second commit - edit all the operator pages.

Fixes #32497

For this issue, update the title, description, and opening paragraph to include the keyword, name of operators, the symbols, and the term used for each operator. Where the opening was a list of the operators, convert that to a paragraph so it shows in search results. Also, do a general edit pass.

For review, checking the opening two paragraphs should be sufficient. 

Also check modified pages for open issues. Fix the following:

#32630 - see Language reference [lambda expressions attributes](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/lambda-expressions?branch=pr-en-us-32704#attributes).
#26112 - see Language reference operators See [operator associativity](https://review.learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/?branch=pr-en-us-32704#operator-associativity).

### CODE REVIEW

File: docs/csharp/language-reference/operators/patterns.md [hunk 1]
```diff
@@ -12,28 +12,9 @@ helpviewer_keywords:
...
...attern](#declaration-and-type-patterns) checks the run-time type of an expression (introduced in C# 9.0). The [constant pattern](#constant-pattern) tests if an expression result equals a specified constant. The [relational patterns](#relational-patterns) compares an expression result with a specified constant (introduced in C# 9.0). [Logical patterns](#logical-patterns) test if an expression matches a logical combination of patterns (introduced in C# 9.0). The [Property pattern](#property-pattern) tests if an expression's properties or fields match nested patterns. [Positional patterns](#positional-pattern)deconstruct an expression result and test if the resulting values match nested patterns. The [List patterns](#list-patterns) tests if a sequence's elements match corresponding nested patterns (introduced in C# 11). The [`var` pattern](#var-pattern): to match any expression and assign its result to a declared variable. The [Discard pattern](#discard-pattern) matches any expression. [Logical](#logical-patterns), [property](#property-pattern), [positional](#positional-pattern), and [list](#list-patterns) patterns are *recursive* patterns. That is, they can contain *nested* patterns.
```
Comment (pkulikov) [inline docs/csharp/language-reference/operators/patterns.md:17]: Combined all in one paragraph and without listing, this te

**Comment by BillWagner** [comment]
@pkulikov 

> could you please elaborate on the necessity of this

The most impactful motivation for this set of changes is [this query](https://www.google.com/search?q=C%23+if+statement).  

> To me, dropping list formatting reduces the readability. I wonder what has made such a trade-off worth it.

I'll take another look on the review site. The first lists I saw were small, and it worked well. I'll give another look at the longer ones. I think for the longer lists, I'll add the keywords in the first paragraph and keep the list for readability. That might be the best presentation.

**Comment by pkulikov** [comment]
@BillWagner I see, thank you for the query example.

**Comment by BillWagner** [comment]
> I see, thank you for the query example.

@pkulikov And thanks for pointing out the readability issue. I think this next version is much easier to read.

**Comment by pkulikov** [comment]
@BillWagner yes, the updated version reads easier, thank you!

**Review by IEvangelist — APPROVED** [review]
This is great, well done! I should work SEO updates for some of the fundamentals too.

**Pull request merged**